examples/webgl_loader_gltf_transmission.html

```diff
@@ -51,15 +52,21 @@
 						texture.dispose();
 						pmremGenerator.dispose();
 
-						render();
-
 						// model
 
-						const loader = new GLTFLoader().setPath( 'models/gltf/' );
-						loader.load( 'TransmissionTest.glb', function ( gltf ) {
+						const loader = new GLTFLoader().setPath( 'models/gltf/IridescentDishWithOlives/glTF/' );
+						loader.load( 'IridescentDishWithOlives.gltf', function ( gltf ) {
```

Comment:
To reduce the number of files tracked here, what do you think of including a GLB version of this instead?

[IridescentDishWithOlives.glb.zip](https://github.com/mrdoob/three.js/files/6798790/IridescentDishWithOlives.glb.zip)

^Draco compressed, which reduces it from 6MB to 5MB, but I'm happy to remove that compression if we'd rather keep this example simpler.

Reply:
+1

Reply:
Updated version, with today's updates to https://github.com/KhronosGroup/glTF-Sample-Models/pull/319.

[IridescentDishWithOlives.glb.zip](https://github.com/mrdoob/three.js/files/6803163/IridescentDishWithOlives.glb.zip)

I removed the compression for now, until I can sort out https://github.com/donmccurdy/glTF-Transform/issues/286.

Reply:
I prefer glb for the example but I was just too lazy to convert the gltf file to a combined glb. Replaced, thanks!